package.json

```diff
@@ -4,11 +4,13 @@
   "description": "Make travel and accommodation easy and convenient.",
   "main": "index.js",
   "scripts": {
-    "start": "nodemon --exec babel-node src/index.js",
     "test": "NODE_ENV=test npm run dropTables && NODE_ENV=test sequelize db:seed:all && NODE_ENV=test mocha --require @babel/polyfill --require @babel/register ./src/tests/*",
     "coverage": "nyc npm test && nyc report --reporter=text-lcov | coveralls",
     "createTables": "npm run dropTables && sequelize db:seed:all",
-    "dropTables": "sequelize db:migrate:undo:all && sequelize db:migrate"
+    "dropTables": "sequelize db:migrate:undo:all && sequelize db:migrate",
+    "build": "babel src --out-dir dist --source-maps inline --copy-files",
+    "start": "node dist/index.js",
```

Comment:
Looks like your start script will not start the app locally without a user explicitly running the build script. Have your:
- `start` script run the build script then also start the app using nodemon instead of just node (so as to refresh the app when changes are made)
e.g.
`npm run build && nodemon dist/index.js`

Reply:
Hi @MuhanguziDavid , please check again. The issue is fixed.